src/commands/integrations/flows/listen.test.ts

```diff
@@ -1,13 +1,189 @@
+>;
+
+const createWebhookFlow = (id: string, name: string): IntegrationFlowNode => ({
+  id,
+  name,
+  stableKey: `${name.toLowerCase().replace(/\s/g, "-")}-key`,
+  description: `${name} description`,
+  testUrl: `https://test.prismatic.io/test/${id}`,
+  trigger: {
+    action: {
+      isPollingTrigger: false,
+      scheduleSupport: ActionScheduleSupport.Valid,
+      component: { key: "webhook-triggers" },
+    },
+  },
+});
+
+const createPollingFlow = (id: string, name: string): IntegrationFlowNode => ({
+  id,
+  name,
+  stableKey: `${name.toLowerCase().replace(/\s/g, "-")}-key`,
+  description: `${name} description`,
+  testUrl: `https://test.prismatic.io/test/${id}`,
+  trigger: {
+    action: {
+      isPollingTrigger: true,
+      scheduleSupport: ActionScheduleSupport.Valid,
+      component: { key: "salesforce" },
+    },
+  },
+});
+
+// Response builders for GraphQL operations
+const buildGetIntegrationFlowsResponse = (
+  flows: IntegrationFlowNode[],
+): { data: GetIntegrationFlowsQuery } => ({
+  data: {
+    integration: {
+      flows: {
+        nodes: flows,
+        pageInfo: { hasNextPage: false, endCursor: null },
+      },
+    },
+  },
+});
+
+const buildGetExecutionsResponse = (
+  executions: Array<{
+    id: string;
+    endedAt?: string | null;
+    requestPayloadUrl?: string;
+    responsePayloadUrl?: string;
+  }>,
+): { data: GetExecutionsQuery } => ({
+  data: {
+    executionResults: {
+      nodes: executions.map((e) => ({
+        id: e.id,
+        endedAt: e.endedAt ?? null,
+        requestPayloadUrl: e.requestPayloadUrl ?? "",
+        responsePayloadUrl: e.responsePayloadUrl ?? "",
+      })),
+    },
+  },
+});
+
+const buildGetPolledExecutionResponse = (
+  execution: {
+    id: string;
+    endedAt?: string | null;
+    stepResults?: Array<{ id: string; stepName: string; resultsUrl: string }>;
+  } | null,
+): { data: GetPolledExecutionQuery } => ({
+  data: {
+    executionResult: execution
+      ? {
+          id: execution.id,
+          endedAt: execution.endedAt ?? null,
+          stepResults: { nodes: execution.stepResults ?? [] },
+        }
+      : null,
+  },
+});
+
+const buildTestIntegrationFlowResponse = (
+  executionId: string,
+): { data: TestIntegrationFlowMutation } => ({
+  data: {
+    testIntegrationFlow: {
+      testIntegrationFlowResult: {
+        execution: { id: executionId },
+      },
+      errors: [],
+    },
+  },
+});
+
+const buildUpdateListeningModeResponse = (
+  errors: Array<{ field: string; messages: string[] }> = [],
+): { data: UpdateIntegrationFlowListeningModeMutation } => ({
+  data: {
+    updateWorkflowTestConfiguration: { errors },
+  },
+});
+
+const server = setupServer(
+  api.mutation("UpdateIntegrationFlowListeningMode", () =>
+    HttpResponse.json(buildUpdateListeningModeResponse()),
+  ),
+);
 
 describe("ListenCommand", () => {
+  beforeAll(() => {
+    server.listen({ onUnhandledRequest: "bypass" });
```

Comment:
Given how much we're mocking in this file should we consider erroring on unhandled requests? Or are there some that are expected still as real live requests?